.changeset/new-spiders-sparkle.md

```diff
@@ -0,0 +1,6 @@
+---
+'@finos/legend-application-studio': patch
+'@finos/legend-graph': patch
+---
```

Comment:
could we separate '@finos/legend-application-studio' and  '@finos/legend-graph' into 2 changesets? 
Leave the description of one that contains '@finos/legend-graph' empty

Reply:
the change in studio should also not be patach but minor. thanks